File: resistor-color-duo/resistor_color_duo.py
def value(colors):
    COLORS = [
        'black',
        'brown',
        'red',
        'orange',
        'yellow',
        'green',
        'blue',
        'violet',
        'grey',
        'white',
    ]
    return 10 * COLORS.index(colors[0]) + COLORS.index(colors[1])

File: javascript/resistor-color-duo/resistor-color-duo.js
//
// This is only a SKELETON file for the 'Resistor Color Duo' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const decodedValue = ([tens, ones]) => {
    return 10 * COLORS.indexOf(tens) + COLORS.indexOf(ones)
};

export const COLORS = ['black', 'brown', 'red', 'orange', 'yellow', 'green', 'blue', 'violet', 'grey', 'white'];
